src/components/remotecontrol/sliderContainer.ts

```diff
@@ -0,0 +1,60 @@
+import { playbackManager } from 'components/playback/playbackmanager';
```

Comment:
Why not `volumeSlider`? `sliderContainer` is for something more generic.

Reply:
Yeah, you're right, I changed the name.